milli/src/update/settings.rs

```diff
@@ -118,14 +110,10 @@ impl<'a, 't, 'u, 'i> Settings<'a, 't, 'u, 'i> {
             distinct_field: Setting::NotSet,
             synonyms: Setting::NotSet,
             primary_key: Setting::NotSet,
-            max_positions_per_attributes: None,
+            indexer_config,
         }
     }
 
-    pub fn log_every_n(&mut self, n: usize) {
-        self.log_every_n = Some(n);
-    }
-
     pub fn reset_searchable_fields(&mut self) {
         self.searchable_fields = Setting::Reset;
     }
```

Comment:
Why don't you remove those functions too?

Reply:
they are not part of the `IndexerConfig`, we could create another struct to hold them, and have it passed to the `Settings` constructor, like we do for the `IndexDocumentsConfig`, but that seemed out of scope for this PR.

Reply:
Good point!